### PR TITLE
https://issues.redhat.com/browse/ACM-8893 CLC intro fixes

### PR DESCRIPTION
**Pull request by oafischer**

### CODE REVIEW

File: clusters/cluster_lifecycle/create_proxy_env.adoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ After you add the previous values, the settings are applied to your clusters.
 [#resources-creating-cluster-proxy]
 == Additional resources
 
-- See xref:../cluster_lifecycle/create_intro.adoc#creating-a-cluster[Creating a cluster] to select your provider.
+- See xref:../cluster_lifecycle/create_intro.adoc#create_intro[Cluster creation introduction] to select your provider.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_proxy_env.adoc:48]: Broken link
Screenshot 2023-12-18 at 1 56 30 PM

File: clusters/cluster_lifecycle/single_node_ocp.adoc [hunk 2]
```diff
@@ -24,7 +24,7 @@ Learn how to deploy a single {sno} cluster. You can also test the procedure of p
 [#sno_create]
 == Create a {sno} cluster
 
-To create a {sno} cluster, complete the following steps in the {product-title-short} console for creating a cluster on one of the providers that are supported to run a {sno} cluster in xref:../cluster_lifecycle/create_intro.adoc#creating-a-cluster[Creating a cluster]:
+To create a {sno} cluster, complete the following steps in the {product-title-short} console for creating a cluster on one of the providers that are supported to run a {sno} cluster in xref:../cluster_lifecycle/create_intro.adoc#create_intro[Cluster creation introduction]:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/single_node_ocp.adoc:27]: This link is fine, but there is a broken link in the file
Screenshot 2023-12-18 at 1 57 24 PM

**Review by dockerymick — APPROVED** [review]
Thanks for the updates Oliver. There are a couple of broken links. I trust that you will fix them haha. Approving

**Comment by oafischer** [comment]
Thanks for catching those links @dockerymick, should all be fixed now!

**Pull request merged**